tools/screenshots/for-events.json

```diff
@@ -0,0 +1,84 @@
+[
+    {
+        "recipient_type": "channel",
+        "channel": "Session 2 Keynote",
+        "color": "#C6A8AD",
+        "topic": "slides and presentation",
+        "users": {
+            "Dal Kim": "static/images/user_avatars/DalKim.jpg",
```

Comment:
Since the avatars are only used for this tool, should we have them live in `tools/screenshots/user_avatars/` instead? I feel like that scopes them more nicely than putting them in `static/images`, which might make it look like they would be used in the actual website directly.

Also, it might be a good idea to just have the basename, `DalKim.jpg`, be in these data structures, and the code itself be responsible for adding the path; that would greatly simplify changing where the avatars live in the user.

Reply:
I guess I'm also not sure we need the name=>avatar mappings be in these individual JSON files at all; could they just be in `thread-screenshot.js`, if the avatars for each user are always the same?

And then this structure could be just `"Dal Kim": {},`, with the idea being the `{}` would be a space for user configuration options in the future (like if we need to set a role or something) but would be just discarded by the script for now.

Reply:
ya, that would make it really clean.